tools/waveica/waveica_wrapper.R

```diff
@@ -10,8 +10,12 @@ waveica <- function(
 ) {
 
     # get input from the Galaxy, preprocess data
-    data <- read.csv(data, header = TRUE, row.names = "sample_name")
-    data <- preprocess_data(data)
+    data <- read.csv(data, header = TRUE, row.names = "sampleName")
```

Comment:
I think relying on name is better than on order, but it is also of course a week spot. Maybe we could supply the column names as arguments to the tool? I know that this is a bit cumbersome, but it would also make it possible to use different input formats etc in the future.

Reply:
I agree with you both. Column names or their order aren't quite reliable; providing column names is more definitive but asking for unnecessary user input. Would we expect our metadata tool to produce tables with inconsistent schema, though? Maybe we address this question once we have that part of the workflow?